src/Classes/ServiceAPI/MyRadio_Podcast.php

```diff
@@ -858,17 +858,12 @@ public function convert()
      */
     public static function getAllPodcasts($num_results = 0, $page = 1, $includeSuspended = true)
     {
-        $query = "SELECT podcast_id FROM uryplayer.podcast
-                  ORDER BY submitted DESC ";
-
-        if (!includeSuspended) {
-            $query .= "WHERE suspended = false ";
-        }
-
+        $where .= !$includeSuspended ? 'WHERE suspended = false ': '';
         $filterLimit = $num_results == 0 ? 'ALL' : $num_results;
         $filterOffset = $num_results * $page;
+        $query = "SELECT podcast_id FROM uryplayer.podcast " . $where 
```

Comment:
Perhaps use some string interpolation? No need to worry about spaces in the extra bits that way
```suggestion
        $query = "SELECT podcast_id FROM uryplayer.podcast $where ORDER BY submitted DESC OFFSET $filterOffset LIMIT $filterLimit";
```